doc(trace-view): Missing trace can be caused by sample rate (#3988)
getsentry/sentry-docs: src/docs/product/sentry-basics/tracing/trace-view.mdx

In addition, broken subtraces can occur when Sentry receives an error for a trace, but the corresponding transaction does not exist. In this case, the error will not appear on the trace view and the trace navigator won't appear on the **Issue Details** page. In these cases you can click "Open In Discover" to see all the events in the trace.

Broken subtraces may be caused by:

> - Sample rates may cause a transaction to be dropped by the SDK, so not all events will be sent to Sentry.

Reviewer: ```suggestion
- Sample rates may cause a transaction to not be sent by the SDK, so not all errors will have an associated transaction sent to Sentry.
```

Reviewer 2: Tightening up a bit. If "in Sentry" changes the meaning, then stick with what Will suggested.

```suggestion
- Sample rates may prevent the SDK from sending a transaction, so not all errors will have an associated transaction in Sentry.
```
